database.cc

```diff
@@ -1331,7 +1331,7 @@ future<> memtable_list::request_flush() {
 }
 
```

Comment:
Is this only for memtables? If so, the pull request description, patch subjects, and variable names should mention it to avoid confusion.

Reply:
It is so probably it's better to update the names of the counters and the other things mentioned by Avi.

Reply:
Updated commit and PR descriptions. Added "memtable_" prefix to variable and counter names.